salt/utils/decorators/__init__.py

```diff
@@ -23,6 +23,10 @@
 # Import 3rd-party libs
 from salt.ext import six
 
+IS_WINDOWS = False
+if getattr(sys, 'getwindowsversion', False):
```

Comment:
Hrm. I think this might neutralize the gains we get from the memoization that we get in `salt.utils.platform.is_windows` and that would not be great. Is there any way to achieve this without doing OS detection?

Reply:
Using `salt.utils.platform.is_windows` was throwing some memoize errors and the test suite wouldn't even start.

We need to set `posix=False` when doing the shlex_split on Windows, otherwise it removes all the `\` in the file path to the batch script that is run. So, I kind of need to detect Windows. I didn't want to import another library, so I went with this method. I could import sys and do `if sys.name == 'nt'` or something... if that makes a difference.

Reply:
After thinking about this for a bit, I think this is fine. This shouldn't be per-invocation so we'll only get the overhead on import.